### PR TITLE
docs: remove strflocaltime from troubleshooting.md

### LINKED ISSUE #4957: [Docs]: remove strflocaltime from troubleshooting
### Is there an existing issue already for your request/idea?

- [X] I have searched for an existing issue, and could not find anything. I believe this is a new documentation enhancement to be evaluated.

### What problem in the existing documentation this issue aims to solve?

The command:
```
kubectl logs -n <NAMESPACE> <CLUSTER>-<N> | \
  jq -r 'select(.logger=="postgres") | [(.ts|strflocaltime("%Y-%m-%dT%H:%M:%S %Z")), .record.message] | @csv'
```

is not working
`jq: error (at <stdin>:5487): strftime/1 requires parsed datetime inputs
`

### Describe what additions need to be done to the documentation

from what @jsilvela told me, this was needed when the kubectl logs where reporting .ts in epoch.
Since this is not the case any more I would remove the strflocaltime.

Then the logs .ts are reportet in UTC and I didn't find a way to change the output to use the local timezone.
Let me know if you have some ideas otherwise I would just remove the strflocaltime.

### Describe what pages need to change in the documentation, if any

remove strflocaltime

### Describe what pages need to be removed from the documentation, if any

troubleshooting.md

### Additional context

please assign this issue to me

### Backport?

Yes

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by smiyc**
closes #4957

Signed-off-by: Daniel Chambre smiyc@pm.me

### CODE REVIEW

**Review by jsilvela — CHANGES_REQUESTED** [review]
The change should work ... however ... sometimes we do see UNIX-style timestamps in the logs.
I think it would be even better if you added an extra paragraph with the original command in case you're getting UNIX timestamps.

**Comment by jsilvela** [comment]
BTW we should not see UNIX timestamps, but the K8s logging is very messy, so there may be paths that don't do RFC3339...

**Review by smiyc — COMMENTED** [review]
> BTW we should not see UNIX timestamps, but the K8s logging is very messy, so there may be paths that don't do RFC3339... 

like so?

File: docs/src/troubleshooting.md [hunk 1]
```diff
@@ -338,7 +338,14 @@ kubectl logs -n <NAMESPACE> <CLUSTER>-<N> | \
...
 
-The following example also adds the timestamp in a user-friendly format:
+The following example also adds the timestamp:
+
+```shell
+kubectl logs -n <NAMESPACE> <CLUSTER>-<N> | \
+  jq -r 'select(.logger=="postgres") | [.ts, .record.message] | @csv'
+```
+
+If the timestamp is displayed in Unix Epoch time, you can convert it to a user-friedgly format:
```
Comment (jsilvela) [inline docs/src/troubleshooting.md:348]:
```suggestion
If the timestamp is displayed in Unix Epoch time, you can convert it to a user-friendly format:
```

**Comment by jsilvela** [comment]
Exactly like you did (without the typo though ;) )

**Review by jsilvela — APPROVED** [review]

**Review by mnencia — APPROVED** [review]

**Pull request merged**